clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -235,6 +235,7 @@ def err_drv_invalid_omp_target : Error<"OpenMP target is invalid: '%0'">;
 def err_drv_invalid_sycl_target : Error<"SYCL target is invalid: '%0'">;
 def err_drv_sycl_target_conflict : Error<"The option -fsycl-targets conflicts with -fsycl-link-targets">;
 def err_drv_sycl_add_link_conflict : Error<"The option -fsycl-link-targets conflicts with -fsycl-add-targets">;
+def err_drv_sycl_link_link_targets_conflict : Error<"The option -fsycl-link-targets conflicts with -fsycl-link">;
```

Comment:
Oh, could you please clarify what the difference between `-fsycl-link-targets` and `-fsycl-link`? I assume in `-fsycl-link-targets` you need additionally pass device targets. Any other differences?

Reply:
-fsycl-link-targets is exclusively used for AOT compilation.  You supply the target triple of what you want to extract from the given object.  Compilation will then stop after the device binary is acquired.  That file is then used during a separate compilation step.

-fsycl-link takes a regular device compilation and instead of stopping at the device binary, it is wrapped.